test/show.jl

```diff
@@ -1,19 +1,103 @@
 module TestShow
     using Base.Test, DataFrames
 
-    df = DataFrame(A = 1:3, B = ["x", "y", "z"])
+    # In the future newline characte \n should be added to this test case
+    df = DataFrame(A = 1:4, B = ["x\"", "∀ε⫺0: x+ε⫺x", "z\$", "ABC"],
+                   C = Float32[1.0, 2.0, 3.0, 4.0])
+    srand(1)
+    df_big = DataFrame(rand(50,50))
+
+    refstr = """
+    4×3 DataFrames.DataFrame
+    │ Row │ A │ B             │ C   │
+    ├─────┼───┼───────────────┼─────┤
+    │ 1   │ 1 │ x\"            │ 1.0 │
```

Comment:
I suppose the fact that vertical lines are not aligned is a bug elsewhere? Then better leave a `TODO` somewhere to make it clear.

Reply:
They are aligned when the string is printed, but `"` needs to be escaped in string literal which breaks alignment in the code.

Reply:
Ah, of course!

Reply:
Actually there is a problem - in the next line `│ 2   │ 2 │ ∀ε⫺0: x+ε⫺x │ 2.0 │` which is not aligned properly and it is a TODO do be added for `getmaxwidths` function. Sorry for confusion